plugins/csp-lod-bodies/src/Plugin.hpp

```diff
@@ -113,6 +125,12 @@ class Plugin : public cs::core::PluginBase {
       std::string mActiveImgDataset; ///< The name of the currently active image data set.
       std::map<std::string, Dataset> mDemDatasets; ///< The data sets containing elevation data.
       std::map<std::string, Dataset> mImgDatasets; ///< The data sets containing image data.
+      cs::utils::DefaultProperty<BRDF> mBrdfHdr{
+          BRDF{"../share/resources/shaders/brdfs/lambert.glsl", {{"$rho", 1.0f}}}};
+      cs::utils::DefaultProperty<BRDF> mBrdfLight{
```

Comment:
I find the name `mBrdfLight` a bit confusing. Maybe `mBrdfNonHdr` would be easier to understand? If you agree, this must be changed in various places :smile: